pkg/util/ansible_execution.go

```diff
@@ -127,7 +128,7 @@ func AnsibleExecution(
 		ansibleEEMounts.Mounts = append(ansibleEEMounts.Mounts, sshKeyMount)
 		ansibleEEMounts.Mounts = append(ansibleEEMounts.Mounts, inventoryMount)
 
-		ansibleEE.Spec.ExtraMounts = []storage.VolMounts{ansibleEEMounts}
+		ansibleEE.Spec.ExtraMounts = append(extraMounts, []storage.VolMounts{ansibleEEMounts}...)
```

Comment:
Merge `extraMounts` into the existing `ansibleEE.Spec.ExtraMounts` like the sshKeyVolme etc.

Reply:
Looks good